draft/0004-experience-api.md

```diff
@@ -345,25 +376,28 @@ The verification APIs (APIs start with `/api/experience/verification`) are a set
 
 #### Social/SSO
 
-- `POST /api/experience/verification/{social|sso}/:connectorId/authorization-url`
+- `POST /api/experience/verification/{social|sso}/:connectorId/authorization-uri`
 
-  Generate an authorization URL for the user to authenticate with a third-party social or enterprise identity provider.
+  Generate an authorization URI for the user to authenticate with a third-party social or enterprise identity provider.
 
   - A social/SSO verification record will be created in the interaction.
   - The unique `verificationId` will be generated for the verification record.
   - A `redirectUrl` will be generated for the user to authenticate with the social/SSO provider.
 
-- `POST /api/experience/verification/{social|sso}/:connectorId/authenticate`
+- `POST /api/experience/verification/{social|sso}/:connectorId/verify`
 
   Authenticate the user with the social/SSO provider and identify the user with the authenticated social/SSO identity.
 
   - The social/SSO verification record will be verified in the interaction.
+  - The `verificationId` is required to retrieve the social/SSO verification record from the interaction.
+  - For Google one tap sign-in, the `verificationId` can be omitted. A new verification record will be created.
 
   Request body:
 
-  | Field | Type                   | Description                                | Required |
-  | ----- | ---------------------- | ------------------------------------------ | -------- |
-  | data  | Record<string, string> | Callback data from the social/SSO provider | true     |
+  | Field          | Type                   | Description                                | Required |
+  | -------------- | ---------------------- | ------------------------------------------ | -------- |
+  | data           | Record<string, string> | Callback data from the social/SSO provider | true     |
```

Comment:
In other tables, we use Yes/No to indicate whether a field is required. Should align them